### ISSUE TITLE
Pretrained embeddings is a 3D tensor

### ISSUE THREAD

**Issue by emanjavacas** [hipster-philology/pandora#89]
See here:
https://github.com/hipster-philology/pandora/blob/master/pandora/pretraining.py#L170-L177

I believe we should get rid of the brackets around the output in the last line:
```
        return [np.asarray(weights, dtype='float32')]
```

should become:

```
        return np.asarray(weights, dtype='float32')
```

**Comment by mikekestemont** [comment]
I believe this is required for the keras model, but we can move the unsqueezing to there.

**Comment by emanjavacas** [comment]
Yes, make it keras specific please. There are already too many things in the pytorch branch that are awkward because we are adopting keras formatting at a higher level.
